workbench/src/renderer/components/DataDownloadModal/index.jsx

```diff
@@ -196,11 +228,11 @@ class DataDownloadModal extends React.Component {
                   )}
                 />
                 <Form.Check.Label>
-                  {modelName}
+                  {displayNames[modelName]}
```

Comment:
Nice catch to include translations for these. I'm thinking about having to remember to update this list manually if model names change or are added / removed. Since we're using `modelName` here, is it possible to somehow programmatically wrap `modelName` with the `t` designation?

Reply:
\`${t(modelName)}\`

Reply:
No, `t(...)` should always go around a string literal because it's used to extract strings for translation. If it goes around a variable, there's no way to know what strings need to be translated.